src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -47,19 +49,22 @@
 
     private ObservableList<TurboIssue> issuesToDisplay = null;
 
-    protected HBox panelMenuBar;
-    protected Text nameText;
-    protected HBox nameBox;
-    protected HBox menuBarNameArea;
-    protected HBox menuBarCloseButton;
-    protected Label renameButton;
-    protected Label closeButton;
-    protected PanelNameTextField renameTextField;
+    public HTPanelMenuBar panelMenuBar;
+    private Text nameText;
+    public HBox nameBox;
+    public HBox menuBarNameArea;
+    public HBox menuBarCloseButton;
+    public HBox menuBarConfirmButton;
+    public HBox menuBarUndoButton;
+    public Label renameButton;
+    public Label closeButton;
```

Comment:
We are moving back to the same problem then. If we move all of FilterPanel's components to PanelMenuBar, PanelMenuBar will become messy (just like the original FilterPanel). I thought PanelMenuBar is only used for rename events and for manipulating it and its children in case of a rename. We still want all of the CSS initialisations in FilterPanel.